src/app/_components/SmartImage.tsx

```diff
@@ -0,0 +1,86 @@
+'use server'
```

Comment:
we need to be explicit to run on the server because we use fs and path which exclusively run in the node env

Reply:
Basically makes sure it runs server side. For example, in `GovernanceCalendarCard` (`use client` component) we import `Card` component which uses `SmartImage`. if we don't explicilty write `use server` in the `SmartImage` component, it will run on the client, and in that case we get an error because we use `fs` which can't run in the browser.